src/main/java/dk/cngroup/lentils/controller/FinalPlaceController.java

```diff
@@ -22,31 +24,35 @@
     private static final String REDIRECT_FINALPLACE_FORM = "redirect:/admin/finalplace/";
 
     private FinalPlaceService finalPlaceService;
+    private ModelMapperWrapper mapper;
 
     @Autowired
-    public FinalPlaceController(final FinalPlaceService finalPlaceService) {
+    public FinalPlaceController(final FinalPlaceService finalPlaceService,
+                                final ModelMapperWrapper modelMapperWrapper) {
         this.finalPlaceService = finalPlaceService;
+        this.mapper = modelMapperWrapper;
     }
 
     @GetMapping(value = "/")
     public String finalPlace(final Model model) {
-        model.addAttribute("finalPlace", finalPlaceService.getFinalPlace());
+        model.addAttribute("finalPlace", mapper.map(finalPlaceService.getFinalPlace(), FinalPlaceFormDTO.class));
```

Comment:
OPTIONAL: Tady bych to taky ulozil do promenne a az tu pak poslal do metody addAttribute kvuli citelnosti.